pkgs/tools/security/sops/default.nix

```diff
@@ -17,6 +17,12 @@ buildGoModule rec {
 
   ldflags = [ "-s" "-w" "-X github.com/getsops/sops/v3/version.Version=${version}" ];
 
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --cmd sops --bash <($out/bin/sops --generate-bash-completion)
```

Comment:
I think the `--generate-bash-completion` is not meant to generate a completion script, instead it generates the completion words:
```console
$ sops --generate-bash-completion
exec-env
exec-file
publish
keyservice
⋮
```
The right solution could be to include a script that uses that flag, similar to:
https://gitlab.archlinux.org/archlinux/packaging/packages/sops/-/blob/9821ec9b759bfb92130e4ccd9e99b48a4ace451d/bash_autocomplete